spikeinterface/comparison/basecomparison.py

```diff
@@ -15,13 +44,16 @@ class BaseComparison:
       * sorting names
       * delta_time to delta_frames
     """
-
-    def __init__(self, sorting_list, name_list=None, delta_time=0.4,  # sampling_frequency=None,
+    def __init__(self, sorting_list, name_list=None, delta_time=0.4,
                  match_score=0.5, chance_score=0.1, n_jobs=-1, verbose=False):
-
         self.sorting_list = sorting_list
         if name_list is None:
-            name_list = ['sorting{}'.format(i + 1) for i in range(len(sorting_list))]
+            name_list = ['sorting{}'.format(i + 1)
+                         for i in range(len(sorting_list))]
```

Comment:
This should be in BaseComparison

Reply:
No, because in case of TemplateComparison they will be called `session`